ts/Extensions/Annotations/Annotations.ts

```diff
@@ -1796,10 +1796,20 @@ chartProto.callbacks.push(function (
 
                             // Annotation not connected to any xAxis -
                             // add new row.
-                            if (xAxisIndex === -1) {
-                                const newRow: any = new Array(event.dataRows[0].length);
+                            let isPie, i;
+                            const series = chart.series;
 
-                                newRow.fill('');
+                            for (i = 0; i < series.length; i++) {
+                                isPie = series[i].is('pie') === false;
```

Comment:
What if we have mixed types? For example:
```js
series: [{ type: 'column' }, { type: 'pie' }, { type: 'column' }]
```

And actually it's not `isPie` but probably `isNotPie`? :)

Reply:
This is a very good question that I also came across while creating this table and has not been discussed before. I think a little redesign of this table would be necessary here, as it may turn out to be less readable (this is just my opinion). It would be great to hear the opinions of others.

isNotPie, agree 👍

Reply:
What I mean is your `for () { ... }` is always setting `isPie` flag only for the last series :) It's no different from `const isPie = series[series.length - 1].is('pie') === false;`. The last item in `for()` has the highest priority.

Anyway, can we focus in this PR in supporting IE11? If there's something else we should discuss or rethink, then could you create separate issue? Thanks! :)

Reply:
Right, I'll create a new ticket for this and improve this logic there.